### DISCUSSION TITLE
Adding Node to Cluster

### DISCUSSION THREAD

**Discussion by ALIEZZ1984** [rabbitmq/cluster-operator#1636] [General]
During the process of configuring the cluster with IPv6, we've encountered some unexpected hurdles.

We are getting the below error when i try to add my node to a cluster. See error below.

I have confirmed that i have access to all the required ports and can ping the primary node via IP and hostname.

Do i need to do any kind of modifications or configuration since im using IPv6 instead of IPv4.

Any suggestions would be appreciated.
![thumbnail_image](https://github.com/rabbitmq/cluster-operator/assets/169689982/2018c544-b252-436a-9c0e-55aa46a1f93c)

**Comment by mkuratczyk** [comment]
You didn't provide any information, so I'm not sure how you expect anyone to help you. At a minimum:
- definition YAML
- server logs
- RabbitMQ version

**Comment by ALIEZZ1984** [comment]
Version of RabbitMQ - 3.12.8
ash-genalab304 is the primary node, ash-genalab305 is second node.
[rabbit@ash-genalab304.log](https://github.com/rabbitmq/cluster-operator/files/15298039/rabbit%40ash-genalab304.log)
[rabbit@ash-genalab305.log](https://github.com/rabbitmq/cluster-operator/files/15298040/rabbit%40ash-genalab305.log)

**Reply by mkuratczyk** [reply]
You are reporting this to a cluster-operator repository, which is a Kubernetes Operator for RabbitMQ. Looks like you are not using it though, so this discussion is misplaced.

switch to 3.13 - the only version with community support right now (https://www.rabbitmq.com/release-information)
don't use 2-node clusters
if you still need to, start a discussion in the right place: https://github.com/rabbitmq/rabbitmq-server/discussions
